src/site/content/en/identity/passkey-form-autofill/index.md

```diff
@@ -165,6 +165,28 @@ Fetch a challenge from the RP server that is required to call
   replay attacks. Make sure to generate a new challenge on every sign-in attempt 
   and disregard it after a certain duration or after a sign-in attempt fails to 
   validate. Consider it like a CSRF token.
+* **[`allowCredentials`](https://w3c.github.io/webauthn/#dom-publickeycredentialrequestoptions-allowcredentials)**:
+  This property is used to find authenticators eligible for this authentication.
+  Pass an empty array or leave it unspecified to let the browser show an account
+  selector.
+* **[`userVerification`](https://w3c.github.io/webauthn/#dom-publickeycredentialrequestoptions-userverification)**:
+  Set to a `"preferred"` value or omit it as it's the default. This indicates
+  whether a user verification using the device's screen lock is `"required"`,
+  `"preferred"` or `"discouraged"`. Setting to `"preferred"` value requests user
+  verification when the device is capable.
+
+{% Aside 'caution' %}
+
+On a device that doesn't support a biometric sensor (e.g. iMac), setting the
+`userVerification` to `"preferred"` results in no user verification requested to
+the user, immediately returning a credential that contains the [UV (user
+verification)](https://w3c.github.io/webauthn/#authdata-flags-uv) flag `false`.
```

Comment:
We could keep this a bit more generic rather than reference iMacs. Also, it's important to know that even with a fingerprint sensor, you may sometimes not get UV. How about 

"For requests with userVerification set to `preferred`, authenticators may skip the user verification check, for example if the device doesn't have any biometric sensor, the user hasn't set it up (e.g. no enrolled fingerprints), or if the sensor is temporarily unavailable (e.g. laptop running with a closed display lid). The UV bit in the authenticator data of the response always indicates whether user verification was performed."